config/settings.yml

```diff
@@ -13,10 +13,10 @@ timeout:
 deployment: core
 
 core:
-  base_url: https://core.collectionspace.org
+  base_url: https://core-dev.cspace.berkeley.edu
   admin:
-    username: admin@core.collectionspace.org
-    password: Administrator
+    username: students@cspace.berkeley.edu
```

Comment:
Hey Sean, please change this back to 
username: admin@core.collectionspace.org
password: Administrator